### ISSUE TITLE
NullPointerException when ignoring :deprecated-var

### ISSUE THREAD

**Issue by xsc** [clj-kondo/clj-kondo#1195]
**version**
clj-kondo v2021.03.03

**platform**
Native on MacOS Mojave.

**problem**
I'm trying to ignore usages of deprecated vars which throws a `NullPointerException`.

**repro**

*Code*
```clojure
(ns repro)

(defn ^{:deprecated "2.0.0"} f1
  [value]
  [value])

(defn f2
  [value]
  #_{:clj-kondo/ignore [:deprecated-var]}
  (f1 value))
```
(Same problem if I move the ignore form to the top-level right before `f2`.)

*clj-kondo --lint*
```
Unexpected error. Please report an issue.
java.lang.NullPointerException
        at clojure.lang.Numbers.ops(Numbers.java:1068)
        at clojure.lang.Numbers.lte(Numbers.java:255)
        at clj_kondo.impl.findings$ignored_QMARK_.invokeStatic(findings.clj:35)
        at clj_kondo.impl.findings$reg_finding_BANG_.invokeStatic(findings.clj:49)
        at clj_kondo.impl.linters$lint_var_usage.invokeStatic(linters.clj:404)
        at clj_kondo.core$run_BANG_.invokeStatic(core.clj:151)
        at clj_kondo.main$main.invokeStatic(main.clj:117)
        at clj_kondo.main$main.doInvoke(main.clj:113)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clj_kondo.main$_main$fn__15849.invoke(main.clj:136)
        at clj_kondo.main$_main.invokeStatic(main.clj:136)
        at clj_kondo.main$_main.doInvoke(main.clj:134)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clj_kondo.main.main(Unknown Source)
```

**config**
Still throws without config.

**expected behavior**
Neither complain, nor crash. :)

**Comment by borkdude** [comment]
Fixed on master.
